### PR TITLE
QA-336: Bug fix/cleanup successfull recovery

### PR DESCRIPTION
**Pull request by dothebart**
### Scope & Purpose

One failing recovery test would add the working directory of all successfull tests as well resulting in huge reports. 

- [x] :hankey: Bugfix

### CODE REVIEW

**Comment by dothebart** [comment]
To demonstrate the effect, one run will be done with two sabotaged tests.

File: tests/test-definitions.txt [hunk 1]
```diff
@@ -90,7 +90,7 @@ resilience_failover priority=750 cluster -- --dumpAgencyOnError true
 resilience_move priority=600 cluster -- --dumpAgencyOnError true
 resilience_sharddist cluster -- --dumpAgencyOnError true
 
-recovery_cluster priority=2000 cluster buckets=8 -- --test search
```
Comment (jsteemann) [inline tests/test-definitions.txt:93]: Why this change? Is it established that the non-search recovery tests work in cluster?

Comment (dothebart) [reply tests/test-definitions.txt:93]: the cluster suite now does that filtering: 
```
                                       // At the moment only view-tests supported by cluster recovery tests:
                                      ).filter(testname => testname.search('search') >= 0);
```
since I don't think that we should abuse test filtering for this.

**Review by jsteemann — APPROVED** [review]
LGTM
